Registry/registry.new3d_wif

```diff
@@ -18,6 +18,7 @@ rconfig   integer wif_input_opt     namelist,domains       1              1
 
 state    real       qnwfa_gc       i{wif}j      dyn_em      1        Z     i1         "QNWFA"   "water-friendly aerosol  num concentration"   "# kg-1"
 state    real       qnifa_gc       i{wif}j      dyn_em      1        Z     i1         "QNIFA"   "water-friendly aerosol  num concentration"   "# kg-1"
+state    real       qnbca_gc       i{wif}j      dyn_em      1        Z     i1         "QNBCA"   "black carbon   aerosol  num concentration"   "# kg-1"
```

Comment:
@twjuliano 
Tim,
Where do the fields for organic carbon biomass burning and black carbon biomass burning come from? This is only black carbon.

Reply:
@davegill The organic carbon and black carbon biomass burning emissions are 2D fields that are added to the 3D QNWFA and QNBCA scalars, respectively, in mp_thompson.